src/Microsoft.AspNetCore.Mvc.Razor/RazorViewEngineOptions.cs

```diff
@@ -36,6 +36,60 @@ public class RazorViewEngineOptions
         public IList<IFileProvider> FileProviders { get; } = new List<IFileProvider>();
 
         /// <summary>
+        /// Gets the locations where <see cref="RazorViewEngine"/> will search for views.
+        /// </summary>
+        /// <remarks>
+        /// <para>
+        /// The locations of the views returned from controllers that do not belong to an area.
+        /// Locations are composite format strings (see http://msdn.microsoft.com/en-us/library/txafckwd.aspx),
+        /// which may contain following indexes:
```

Comment:
`which may contain the following format items:` (missing "the" and use "format items").